include/internal/libspdm_common_lib.h

```diff
@@ -962,10 +962,12 @@ bool libspdm_verify_psk_finish_req_hmac(libspdm_context_t *spdm_context,
  *
  * This function should be called in KEY_EXCHANGE/PSK_EXCHANGE request generation.
  *
+ * @param  version_count                 Secure version count.
+ * 
  * @return the size in bytes of opaque data supproted version.
  **/
 size_t libspdm_get_opaque_data_supported_version_data_size(
-    libspdm_context_t *spdm_context);
+    libspdm_context_t *spdm_context, uint8_t version_count);
```

Comment:
Why you need version_count as input?
Why not just use spdm_context->local_context.secured_message_version.spdm_version_count ?